contracts/FlowIDTableStaking.cdc

```diff
@@ -473,18 +481,6 @@ pub contract FlowIDTableStaking {
         pub fun unstakeAll() {
             let nodeRecord = FlowIDTableStaking.borrowNodeRecord(self.id)
 
-            // iterate through all their delegators, uncommit their tokens
-            // and request to unstake their staked tokens
-            for delegator in nodeRecord.delegators.keys {
-                let delRecord = nodeRecord.borrowDelegatorRecord(delegator)
-
-                if delRecord.tokensCommitted.balance > 0.0 {
-                    delRecord.tokensUnstaked.deposit(from: <-delRecord.tokensCommitted.withdraw(amount: delRecord.tokensCommitted.balance))
-                }
-
-                delRecord.tokensRequestedToUnstake = delRecord.tokensStaked.balance
-            }
-
```

Comment:
I'm not quite following what this is replaced by?

Reply:
I actually didn't have to add anything to replace this, because it was already replaced. In the `endStaking` function, we check if the node is committed below the minimum, and if they are, it marks all their delegators to unstake.

https://github.com/onflow/flow-core-contracts/blob/josh/many-nodes/contracts/FlowIDTableStaking.cdc#L682